types/lodash/index.d.ts

```diff
@@ -7221,296 +7257,879 @@ declare namespace _ {
         ): T|undefined;
     }
 
-    interface LoDashExplicitWrapperBase<T, TWrapper> {
+    interface LoDashExplicitWrapperBase<T, TWrapper> {
+        /**
+         * @see _.findLast
+         */
+        findLast(
```

Comment:
@andy-ms I think I've found a way to handle the explicit wrappers correctly - instead of having many different types of wrappers (`LodashExplicitArrayWrapper`, `LodashExplicitObjectWrapper`, etc), we should just have one wrapper type: `LodashExplicitWrapper<T>`. Then each individual function would use the `this` parameter to specify what kind of `T` it needs. For example:

```
interface LodashExplicitWrapper<T> {
    findLast<TItem>(
        this: LodashExplicitWrapper<List<TItem> | null | undefined>,
        size?: number,
   ): LodashExplicitWrapper<TItem>;
}
```

However, this will require a complete overhaul of how the lodash wrappers work, so I think it belongs in a separate PR (it's out-of-scope for this one). I can make (yet another) PR for that, but first let me know if you have any concerns about that approach.